test/e2e/config/vsphere.yaml

```diff
@@ -176,6 +178,20 @@ providers:
           # Add cluster templates
           - sourcePath: "../data/shared/main/v1beta1_provider/metadata.yaml"
 
+  - name: vm-operator
+    type: InfrastructureProvider
```

Comment:
Should we also make this an `RuntimeExtensionProvider`? From CAPI perspective, CAPV is our infra provider.

Reply:
Yes, that's my intention